services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+import datetime
+
+from django.db.models import QuerySet
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: datetime or None = None) -> QuerySet:

```

Comment:
datetime is a module, datetime.datetime is a class. Also why do assign in annotation None to None, maybe better directly assign parameter to None as default.